aii-core/src/main/perl/Shellfe.pm

```diff
@@ -657,7 +657,7 @@ sub fetch_profiles
             next;
         }
         my @err = $fh->fetchProfile();
-        if ($err[0] != SUCCESS) {
+        if (! $err[0] || $err[0] == -1) {
```

Comment:
-1 looks like a magic number, can it be replaced by a constant? Or better yet, let's do ```|| $err[0] < 0``` because then in future release we can replace the undef return with -2 and make all errors negative numbers and all successful results positive ones.

cf. [fetchProfile doc](https://github.com/quattor/CCM/blob/master/src/main/perl/Fetch.pm#L125)